daft/expressions/expressions.py

```diff
@@ -347,6 +347,17 @@ def sign(self) -> Expression:
         expr = self._expr.sign()
         return Expression._from_pyexpr(expr)
 
+    def round(self, decimals: int = 0) -> Expression:
+        """The round of a numeric expression (``expr.round(decimals = 0)``)
+
+        Args:
+            decimals: number of decimal places to round to. Defaults to 0.
+        """
+        if decimals is None:
+            decimals = 0
```

Comment:
```suggestion
        assert isinstance(decimals, int)
```

Let's have a type check instead that'll encompass more edge cases than just `None`.